apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -100,6 +120,10 @@ var TopInstructions = React.createClass({
     ttsMarkdownInstructionsUrl:  PropTypes.string
   },
 
+  state:{
+    helpTabVisible: false,
```

Comment:
I might call this something like `helpTabFocused` or `helpTabSelected`. As it's named right now, I might expect the entire tab to be hidden (vs. just the tab contents).